index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <title>Mobile Portfolio</title>
+  <header>
+    <img src="img/My Logo.svg" alt="humburger" class="Logo">
+    <img src="img/Union.svg" alt="humburger" class="Union">
+  </header>
+</head><br><br><br><br>
+
+
+<body>
+  <div id="my-section">
+    <section class="container">
+      <h1>I'm Edgah.<br>
+        Glad to see<br>
+        you!</h1>
+      <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    </section>
+
+
+    <section>
+      <ul class="social-icons">
+        <li>
+          <img src="img/twitter icon.svg">
+        </li>
+        <li>
+          <img src="img/Linkedin icon.svg">
+        </li>
+        <li>
+          <img src="img/Mobile icon.svg">
+        </li>
+        <li>
+          <img src="img/git icon.svg">
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li>
+      </ul>
+    </section><br><br><br><br><br><br><br><br>
+  </div>
```

Comment:
- [OPTIONAL] This is not a requirement for this milestone, But I'll like to suggest to you that following the best practices, you are meant to have this in a `header` tag, this is the website header, and using the correct semantic tag makes the website rank good in SEO and helps website reader understand your website structure well. Also, the hamburger menu is a nav and should be in a `nav` tag. :+1: :+1: :100: :+1: